app/components/job-infrastructure-notification.js

```diff
@@ -19,15 +19,21 @@ export default Component.extend({
   @equal('queue', 'builds.linux') isLegacyInfrastructure: null,
   @equal('queue', 'builds.ec2') isTrustySudoFalse: null,
 
-  @computed('job.startedAt', 'job.config')
-  isTrustyStable(startedAt, config = {}) {
-    if (config.dist === 'trusty' && config.group === 'stable') {
+  @computed('job.startedAt', 'queue', 'job.config')
```

Comment:
I think this should be `job.config.dist` for the last dependent key, but maybe you had that before and it didn’t work?